source/test/TestApp/PlayFabTestMultiUserInstance.cpp

```diff
@@ -23,21 +23,21 @@ namespace PlayFabUnit
 
         const auto& user1ProfileSuccess = std::bind(&PlayFabTestMultiUserInstance::MultiUserProfile1Success, this, std::placeholders::_1, std::placeholders::_2);
         const auto& user1ProfileFailure = std::bind(&PlayFabTestMultiUserInstance::MultiUserProfile1Failure, this, std::placeholders::_1, std::placeholders::_2);
-        (*multiUser1ClientApi)->GetPlayerProfile(profileRequest, user1ProfileSuccess, user1ProfileFailure, customData);
+        multiUser1ClientApi->GetPlayerProfile(profileRequest, user1ProfileSuccess, user1ProfileFailure, customData);
     }
-    void PlayFabTestMultiUserInstance::MultiUserLogin1Failure(const PlayFabError& error, void* customData)
+    void PlayFabTestMultiUserInstance::MultiUserLogin1Failure(const PlayFabError& error, void* /*customData*/)
```

Comment:
Awesome :) I ran into this warning today about unused variables, I'll follow the pattern you have here.

Reply:
FWIW, another option we use in bumblelion is to just mark the parameter unused in the function body
```
UNREFERENCED_PARAMETER(customData); // this may be a windows macro
(void)customData; // equivalent to the macro above
```

another option is to just leave the name unspecified
`MultiUserLogin1Failure(const PlayFabError& error, void*);`

Reply:
My first resolution tried to use UNREFERENCED_PARAMETER.
UNREFERENCED_PARAMETER is not defined on all platforms, and was a compile error when I applied it trivially here.
I think I took one more step to attempt to also define it, and doing so also didn't trivially work, so I opted for commenting the input name as a solution we're already using in this codebase.

As for leaving the name unspecified: yeah, which is what this PR technically does by commenting it.
However, because it's a void*, it can be hard to figure out what it was supposed to be originally, if you remove the name.
It makes it harder to bring it back in a likely future where it's needed again.

---
In reply to: [343273143](https://github.com/PlayFab/XPlatCoreTemplate/pull/39#discussion_r343273143) [](ancestors = 343273143)